app/controllers/employees/rewards/orders_controller.rb

```diff
@@ -0,0 +1,29 @@
+module Employees
+  module Rewards
+    class OrdersController < EmployeesController
+      before_action :set_reward
+      def new
+        @order = Order.new
+      end
```

Comment:
Why did you created method `new` as well as `view` for it if you are not using it anywhere?

Reply:
same as up